core/runtimes/FiberRuntime/index.js

```diff
@@ -23,8 +25,8 @@ FiberRuntime.prototype._close = function (proc) {
   proc.status = Process.Status.CLOSED;
   // console.log('cl' + count);
   this._count--;
-  proc.outports.forEach(function(outPort) {
-    var conn = outPort[1].conn;
+  _.forEach(proc.outports, function (outPort) {
```

Comment:
`outports` and `inports` are now associative arrays, so we have to use `_.forEach`